prospero-common/src/main/java/org/wildfly/prospero/actions/FeaturesAddAction.java

```diff
@@ -255,9 +255,15 @@ public FeaturePackTemplate getFeaturePackRecipe(String featurePackCoord)
             throw ProsperoLogger.ROOT_LOGGER.unableToCreateTemporaryDirectory(e);
         } finally {
             if (galleonEnv != null) {
+                if (ProsperoLogger.ROOT_LOGGER.isDebugEnabled()) {
+                    ProsperoLogger.ROOT_LOGGER.debug("Closing galleon env");
+                }
                 galleonEnv.close();
             }
             if (tempDirectory != null) {
+                if (ProsperoLogger.ROOT_LOGGER.isDebugEnabled()) {
+                    ProsperoLogger.ROOT_LOGGER.debug("Removing temporary folder: " + tempDirectory);
```

Comment:
Sorry for being pedantic, but `debugf` has been used a couple of lines above. Any reason not to use it here?

Reply:
not enough coffee 😂 thanks, I'll update it